lib/liquid/tags/table_row.rb

```diff
@@ -29,7 +29,7 @@ def render(context)
       cols = context.evaluate(@attributes['cols'.freeze]).to_i
 
       result = "<tr class=\"row1\">\n"
-      context.stack do
+      context.stack('tablerowloop', @variable_name) do
```

Comment:
Shouldn't `'tablerowloop'` be frozen here?
```suggestion
      context.stack('tablerowloop'.freeze, @variable_name) do
```